xcp_d/interfaces/bids.py

```diff
@@ -132,6 +131,8 @@ def __init__(self, allowed_entities=None, out_path_base=None, **inputs):
             setattr(self.inputs, k, inputs[k])
 
     def _run_interface(self, runtime):
+        from bids.layout import parse_file_entities
+
```

Comment:
Why move this import down from the top of the file?

Reply:
It wasn't being imported unless I moved it into the function